README.md

```diff
@@ -8,4 +8,32 @@
 > **The source for the latest release can be found at [Releases](https://github.com/edgexfoundry/app-record-replay/releases).**
 
 
-Content TBD
+For latest documentation please visit https://docs.edgexfoundry.org/latest/microservices/application/services/AppRecordReplay/Purpose/
```

Comment:
Note this link is broken until docs are release an 3.1 is `latest`. For testing you can change it to 3.1 manually when you get the Not Found page.